Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,63 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] vs)
+        {
+            LogLevel logLevel = LogLevel.Error;
+
+            if (baseLogger == null)
+                throw new ArgumentNullException(baseLogger.ToString());
+
+            else {
+                message = Convert.ToString(vs); 
```

Comment:
This overwrites the passed in `message` value. The appropriate thing to do here is use the `string.Format` method.
```suggestion
                string newMessage = string.Format(message, vs); 
```